rad_gen.py

```diff
@@ -55,63 +58,112 @@
 ##### 3D IC IMPORTS ##### 
 import src.ic_3d.ic_3d as ic_3d
 
+import src.common.constants as consts
+
 
 rad_gen_log_fd = "rad_gen.log"
 log_verbosity = 2
 cur_env = os.environ.copy()
 
+# ██╗      ██████╗  ██████╗  ██████╗ ██╗███╗   ██╗ ██████╗ 
+# ██║     ██╔═══██╗██╔════╝ ██╔════╝ ██║████╗  ██║██╔════╝ 
+# ██║     ██║   ██║██║  ███╗██║  ███╗██║██╔██╗ ██║██║  ███╗
+# ██║     ██║   ██║██║   ██║██║   ██║██║██║╚██╗██║██║   ██║
+# ███████╗╚██████╔╝╚██████╔╝╚██████╔╝██║██║ ╚████║╚██████╔╝
+# ╚══════╝ ╚═════╝  ╚═════╝  ╚═════╝ ╚═╝╚═╝  ╚═══╝ ╚═════╝ 
+
+
+def init_logger():
+    # Init Logger
+    logger = logging.getLogger(consts.LOGGER_NAME)
+    
+    logger.root.setLevel(logging.DEBUG)
+    # Create Stream Handler
+    stream_handler = logging.StreamHandler()
+    stream_handler.setLevel(logging.DEBUG) # Set this handler to print out everything (debug is lowest level)
+    file_handler = logging.FileHandler(os.path.abspath(f"rad_gen_logging_{rg_ds.create_timestamp()}.log"))
+    file_handler.setLevel(logging.DEBUG)
+    fmt = logging.Formatter('%(message)s')
+    stream_handler.setFormatter(fmt)
+    file_handler.setFormatter(fmt)     
+    logger.addHandler(stream_handler)
+    logger.addHandler(file_handler)
+    return logger
+
 # ██████╗  █████╗ ██████╗      ██████╗ ███████╗███╗   ██╗    ███████╗██╗  ██╗███████╗ ██████╗    ███╗   ███╗ ██████╗ ██████╗ ███████╗███████╗
 # ██╔══██╗██╔══██╗██╔══██╗    ██╔════╝ ██╔════╝████╗  ██║    ██╔════╝╚██╗██╔╝██╔════╝██╔════╝    ████╗ ████║██╔═══██╗██╔══██╗██╔════╝██╔════╝
 # ██████╔╝███████║██║  ██║    ██║  ███╗█████╗  ██╔██╗ ██║    █████╗   ╚███╔╝ █████╗  ██║         ██╔████╔██║██║   ██║██║  ██║█████╗  ███████╗
 # ██╔══██╗██╔══██║██║  ██║    ██║   ██║██╔══╝  ██║╚██╗██║    ██╔══╝   ██╔██╗ ██╔══╝  ██║         ██║╚██╔╝██║██║   ██║██║  ██║██╔══╝  ╚════██║
 # ██║  ██║██║  ██║██████╔╝    ╚██████╔╝███████╗██║ ╚████║    ███████╗██╔╝ ██╗███████╗╚██████╗    ██║ ╚═╝ ██║╚██████╔╝██████╔╝███████╗███████║
 # ╚═╝  ╚═╝╚═╝  ╚═╝╚═════╝      ╚═════╝ ╚══════╝╚═╝  ╚═══╝    ╚══════╝╚═╝  ╚═╝╚══════╝ ╚═════╝    ╚═╝     ╚═╝ ╚═════╝ ╚═════╝ ╚══════╝╚══════╝
 
-def main(args: Optional[List[str]] = None) -> None:
+def main(args: argparse.Namespace | None = None) -> Tuple[ Any | None, rg_ds.Tree]:
     global cur_env
     global rad_gen_log_fd
     global log_verbosity
 
+    # Get logger into main
+    # logger = init_logger()
```

Comment:
Is this intended to be commented out? Perhaps it would be a good CLI option to turn this on / off?